modules/kafka/kafka-dest-driver.c

```diff
@@ -277,7 +362,9 @@ _flush_inflight_messages(KafkaDestDriver *self)
   if (outq_len > 0)
     {
       msg_notice("kafka: shutting down kafka producer, while messages are still in-flight, waiting for messages to flush",
-                 evt_tag_str("topic", self->topic_name),
+
+                 evt_tag_str("topic", self->topic_name->template),
+                 self->fallback_topic_name ? evt_tag_str("fallback_topic", self->fallback_topic_name) : NULL,
```

Comment:
I think the `NULL` here stops the processing of all the `evt_tags` after. Can you check that please? Maybe `evt_tag_str("fallback_topic", self->fallback_topic_name ? : "N/A")` would be better.

Same thing with the other log message of this function.

Reply:
Yes, tested it now. It is ignoring the evt_tag after NULL
@bazsi, In the earlier PR comments, you had told me about using NULL instead of "NULL" string. 
How do I do it the right way(other than keeping fallback_topic at the end)?

Reply:
just include it unconditionally. the value would become "(null)" or something like this if it's null.